packages/react/README.md

```diff
@@ -0,0 +1,20 @@
+# MUD React
+
+React hooks (and more) for building MUD clients.
+
+### useComponentValue
+
+Returns the value of the component for the entity, and triggers a re-render as the component is added/removed/updated.
+
+```
+const position = useComponentValue(entity, Position);
```

Comment:
While building Emojimon, I was thinking about ECS in terms of entity-first, where "entity" is often the thing to key off of, and components are types of values on an entity.

However, I noticed our internal APIs (`setComponent`, `removeComponent`), etc. all have components first, then entities. I wonder if this is because we internally model a component as a mapping of entities <> values.

How do you feel about the position of the arguments here?

Reply:
Yes that's mostly the reason - when thinking of components as tables and entities as rows in that table, it felt more natural to first specify which table to modify and then which row in that table. In general I don't feel strongly about the order of arguments, can totally see how putting entity first is more intuitive when thinking about entities as objects and components as parameters on that object. My main argument for changing the order here to also put component first would be consistency with existing functions.

Reply:
Chatted with @alvrs some more and here's what I got out of the convo:

Data modeling with MUD can be thought of in one of two ways
- Entity-first, where the entity is the focus and data (components/values) are "attached" to an entity. I was thinking about it like this, where my "get component value" operation is more like a "get this value of this key from the entity object"
- Component/Table-first, where it's more like a relational database and the entity ID is a primary key. In the future, we may want more complicated keys (tuples) that would expand past just an entity ID. So asking for a component value is more like "get the value in the component table for this primary key aka entity ID".

Sounds like we want to guide folks towards the latter mental model, esp. as we push further into SQL-like capability.

Will update these hooks!